docs/installation.rst

```diff
@@ -51,18 +51,22 @@ URL paths
 ~~~~~~~~~
 
 Include the required CAS URL endpoints in your project's root ``URLconf``
-with the following::
+in ``urls.py`` with the following::
 
+   from django.urls import include, re_path
+   
    urlpatterns = [
        # ...existing urls...
-       url(r'', include('mama_cas.urls')),
+       re_path(r'', include('mama_cas.urls')),
    ]
 
 This makes the CAS server available at the top level of your project's
 URL (e.g. ``http://example.com/login``). To add a path to the CAS root
 (e.g. ``http://example.com/cas/login``) include the path in the URL regular
 expression::
 
+   from django.urls import include, re_path
+   
```

Comment:
Again, this should be already in your urls.py file, but ok to include it.

Reply:
This isn't as obvious when coming to this guide to the first time without knowing the project nor Python very well. :)